package.json

```diff
@@ -28,22 +34,28 @@
   },
   "homepage": "https://github.com/SortableJS/angular-sortablejs#readme",
   "dependencies": {
-    "@angular/common": "^2.0.0",
-    "@angular/core": "^2.0.0",
-    "@angular/forms": "^2.0.0",
-    "rxjs": "^5.0.0-beta.12",
+    "@angular/common": "2.1.0",
```

Comment:
Cool. Please just make sure to check what version gets actually installed (to avoid 2.1.4) and to test both dev & dist builds after you install typescript.